javascript/lib/node/src/node-http.ts

```diff
@@ -96,7 +96,7 @@ export class NodeRequester implements HttpRequester {
       host: parsedUrl.host,
       path: parsedUrl.pathname
     };
-    if (this.proxyAgent !== undefined) {
+    if (this.proxyAgent.options.path !== undefined) {
```

Comment:
```suggestion
    if (this.proxyAgent !== undefined && this.proxyAgent.options.path !== undefined) {
```

Maybe adjust it like this in order to don't break what the creator of this intended?

Reply:
The way it's written right now, this.proxyAgent will never be undefined. But sure, I can keep that part in.